Sources/FluentBenchmark/Galaxy.swift

```diff
@@ -16,6 +17,9 @@ public final class Galaxy: Model {
     @Children(for: \.$galaxy)
     public var planets: [Planet]
 
+    @Timestamp(key: "deleted_at", on: .delete)
```

Comment:
Could we create a new, separate model for this deleted_at test? `Galaxy` is used in _tons_ of tests and I wanna keep it simple to reduce variables in the test cases.